src/integrations/bluetooth-low-energy/bluetooth-low-energy.service.ts

```diff
@@ -14,19 +14,27 @@ import { ClusterService } from '../../cluster/cluster.service';
 import { NewDistanceEvent } from './new-distance.event';
 import { EntityCustomization } from '../../entities/entity-customization.interface';
 import { SensorConfig } from '../home-assistant/sensor-config';
+import { DeviceTrackerConfig } from '../home-assistant/device-tracker-config';
 import { RoomPresenceDistanceSensor } from '../room-presence/room-presence-distance.sensor';
 import { SchedulerRegistry } from '@nestjs/schedule';
 import { KalmanFilterable } from '../../util/filters';
 import { makeId } from '../../util/id';
 import { DISTRIBUTED_DEVICE_ID } from '../home-assistant/home-assistant.const';
 import * as _ from 'lodash';
 import { DeviceTracker } from '../../entities/device-tracker';
-import { RoomPresenceDeviceTrackerProxyHandler } from '../room-presence/room-presence-device-tracker.proxy';
+import { Sensor } from '../../entities/sensor';
+import { RoomPresenceProxyHandler } from '../room-presence/room-presence.proxy';
 import { BluetoothLowEnergyPresenceSensor } from './bluetooth-low-energy-presence.sensor';
 import { BluetoothService } from '../bluetooth/bluetooth.service';
 
 export const NEW_DISTANCE_CHANNEL = 'bluetooth-low-energy.new-distance';
 
+class deviceCustomizations {
```

Comment:
Instead of defining this custom type I think it would be better to use `Partial<Device>` as the type here, as that would re-use the definitions we already have in the Home Assistant integration. I think it's probably easiest to replace the usages of this class down below and then remove this.

Reply:
Good catch. I can use Device type directly as I have the mandatory fields.